ubuntu/cuda/uptodate.yaml

```diff
@@ -1,14 +1,12 @@
 dockerbuild:
   matrix:
     cuda_version:
-       - "10.1.243"
```

Comment:
Is this because CUDA 10 required Ubuntu 18? I think it would be good to keep this version if possible, so that we have at least 1 image with the previous major version.

Reply:
I'm not sure - I can try adding it back and we will find out!

Reply:
Perfect, thanks!

Reply:
CUDA 10 doesn't work with default GCC in Ubuntu 20: https://github.com/rse-ops/docker-images/issues/48